### PR TITLE
typings update to catch up with current version of react-virtualized

### PR DESCRIPTION
**Pull request by kaoDev**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/bvaughn/react-virtualized>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/react-virtualized/dist/es/List.d.ts [hunk 1]
```diff
@@ -73,60 +90,44 @@ export type ListProps = GridCoreProps & {
...
-        noRowsRenderer: () => null,
-        onRowsRendered: () => null,
-        onScroll: () => null,
-        overscanRowCount: 10,
-        scrollToAlignment: 'auto',
-        scrollToIndex: -1,
-        style: {}
+        autoHeight: false;
+        estimatedRowSize: 30;
+        onScroll: () => {};
```
Comment (Kovensky) [inline types/react-virtualized/dist/es/List.d.ts]: Why not `=> void`?

**Review by Jessidhia — COMMENTED** [review]
I just skimmed through this; I'm curious as to why some shapes are defined as `interface`s and others as `type`s.

Is it up to which ones you want to allow users to augment / `extend`?

File: types/react-virtualized/dist/es/List.d.ts (hunk 1 above)
Comment (kaoDev) [reply types/react-virtualized/dist/es/List.d.ts]: thanks for the hint, that was a copy paste mistake

**Comment by kaoDev** [comment]
the interface was also just copied from the original code, it's now changed to a type for consistency with the rest

**Pull request merged**